tests/html_vue/template-lang.vue

```diff
@@ -13,3 +13,8 @@
     #foo
   .bla
 </template>
+
+<template lang="pug">
```

Comment:
```suggestion
<template lang="unknown">
```

We may support `pug` someday.